Removing the ability to edit the OC container title (#4559)
woocommerce/woocommerce-gateway-stripe: changelog.txt

*** Changelog ***

= 9.8.0 - xxxx-xx-xx =

> * Update - Removes the ability to change the title for the Optimized Checkout payment element, as it is now set to "Stripe" by default

Reviewer: It may be simpler to specify what string we are showing rather than talking about a default.
```suggestion
* Update - Removes the ability to change the title for the Optimized Checkout payment element, and use "Stripe" as the title
```
